packages/common/src/types/filter.ts

```diff
@@ -1,9 +1,4 @@
-import {
-    ConditionalOperator as FilterOperator,
-    ConditionalRule,
-} from './conditionalRule';
```

Comment:
Fixes tsoa routes not being happy with an `enum` alias. export with `as` happens at the end of the file now.